### PR TITLE
Exposed `chunk` RPC API (nearcore 0.4.2+)

### PR DESCRIPTION
**Pull request by frol**
Ref: https://github.com/nearprotocol/nearcore/pull/1524

### CODE REVIEW

File: test/providers.test.js [hunk 1]
```diff
@@ -13,6 +13,18 @@ test('json rpc fetch block info', async () => {
     const provider = new nearlib.providers.JsonRpcProvider(config.nodeUrl);
     let response = await provider.block(1);
     expect(response.header.height).toEqual(1);
+    let sameBlock = await provider.block(response.header.hash);
+    expect(sameBlock.header.height).toEqual(1);
+});
+
+test('json rpc fetch chunk info', async () => {
+    const config = Object.assign(require('./config')(process.env.NODE_ENV || 'test'));
```
Comment (vgrichina) [inline test/providers.test.js]: can we extract this into helper or `before`? Seems like every test is loading config and creating provider with it.

**Comment by frol** [comment]
@vgrichina I expect it will pass once nearcore is deployed with the PR, so in theory, we only need to run CI once more once that happens. I am just proactively creating PRs :)

File: test/providers.test.js (hunk 1 above)
Comment (frol) [reply test/providers.test.js]: Sure, I will fix it.

**Comment by vgrichina** [comment]
> @vgrichina I expect it will pass once nearcore is deployed with the PR, so in theory, we only need to run CI once more once that happens. I am just proactively creating PRs :)

Do you plan to wait until it is deployed on master testnet or on staging? If you only want to wait for staging deploy (or not wait at all), then it's necessary to gate this test by nearcore version.

**Comment by frol** [comment]
I plan to wait for the API hit `master`. I hope it happens soon since without this API Explorer is useless (I cannot get a transactions list from nearcore otherwise).

**Comment by frol** [comment]
@vgrichina The tests pass on the current (already deployed) master. Staging should work fine as well, though it is down at the moment. The CI deployed nodes needs to be updated to include the `chunk` API.

**Pull request merged**